Lib/typing.py

```diff
@@ -659,25 +654,29 @@ class FastConnector(Connection):
 def Union(self, parameters):
     """Union type; Union[X, Y] means either X or Y.
 
-    To define a union, use e.g. Union[int, str].  Details:
+    On Python 3.10 and higher, the | operator
+    can also be used to denote unions in many situations;
```

Comment:
unnecessary clause, unless you have something specific in mind
```suggestion
    can also be used to denote unions;
```

Reply:
The reason I used this wording is you can't use types.UnionType with forward references, but you can with typing.Union. But maybe that doesn't need to be mentioned here